corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -29,7 +29,13 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
                 defer = $.Deferred(),
                 options,
                 menus;
-
+                sessionStorage.formplayerQueryInProgress = true;
```

Comment:
Just checking, is `queryFormplayer` the only route that should trigger loading indicator? For example, I think breadcrumb navigation doesn't hit this so the prominent loading bar wouldn't show. But the current green progress bar does

Reply:
Breadcrumb nav will make a `navigate_menu` request and query formplayer, so the progress bar still gets triggered